src/main/java/Handlers/FileHandler.java

```diff
@@ -0,0 +1,106 @@
+package Handlers;
+
+import Exceptions.AikhsuException;
+import Tasks.*;
+
+import java.io.*;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Handles file operations for saving and loading tasks.
+ * This class provides methods to write tasks to a file and retrieve them when needed.
+ */
+public class FileHandler {
+    private static String FILE_PATH;
```

Comment:
Should this be in camelCase instead? Since it isn't a constant which is immutable.
```suggestion
    private static String filePath;
```